### DISCUSSION TITLE
Tracker for Exchange Issues

### DISCUSSION THREAD

**Discussion by jnthnvctr** [filecoin-project/community#617] [General]
Especially post FVM, there have been mixed reports about challenges folks have had with various exchanges. The primary issues seem to be: 

a) Out of gas issues
b) Lack of support for the FEVM addresses (f4/0x) for deposits and withdrawls. 

This discussion will have a few top level comments - please thread your experiences, and periodically the issues can be summarized / shared with technical PoCs. The fastest way to solve these issues is always to file support issues with the exchange directly (though feel free to also leave a comment in the respective thread so it can be surfaced)

**Comment by jnthnvctr** [comment]
Top level thread for Binance related issues

**Reply by SmartOct** [reply]
Hi
I withdrew some Filecoin (FIL) from my BingX exchange account to my Trust Wallet. However, after the transaction was completed, the FIL balance does not show up in my Trust Wallet. I have checked the transaction history on my Trust Wallet, and it shows that the FIL was transferred successfully. However, the balance remains at 0, and I am unable to withdraw the FIL.

I have attempted to verify my balance using filscan and found that the account balance is also 0. I am unsure of what the issue could be, but I believe that it may be related to the transaction process. I would greatly appreciate it if you could investigate this matter and provide me with a solution as soon as possible.

Thank you for your time and assistance in this matter.

**Reply by SmartOct** [reply]
Here?

**Reply by jnthnvctr** [reply]
@Osamasultan913 can you please comment your issues here / remove the other ones? This is just so we can easily consolidate by exchange the issues folks are facing

**Reply by Osamasultan913** [reply]
Hi
I tried to withdraw my file coins from kraken to Binance and I selected deposit address ERC20 on Binance but while putting address to withdraw from kraken i used FileCoin network and used ERC20 deposit address now my funds are in the blockchain and i cant access them need help 

✓Sent FIL from Kraken to Binance.
✓Chose the Filecoin Network. 
❌Put an ERC20 address.

**Comment by jnthnvctr** [comment]
Top level thread for Coinbase related issues

**Comment by jnthnvctr** [comment]
Top level thread for Kucoin related issues

**Comment by SmartOct** [comment]


**Reply by jnthnvctr** [reply]
Hi! Can you put trustwallet issues in the binance thread?

**Reply by SmartOct** [reply]
Yeah, send me thread link please.

**Reply by jnthnvctr** [reply]
It's above! First comment in this issue :D

**Comment by Osamasultan913** [comment]


**Reply by jnthnvctr** [reply]
Hi can you thread this in the Binance top level [comment](https://github.com/filecoin-project/community/discussions/617#discussioncomment-5490805)?

**Reply by jnthnvctr** [reply]
Just copy and paste in that thread, so we can consolidate and easily share with the Binance tech team

**Comment by olaitantm** [comment]
I FIL forwarded 25 FIL to FIL address OKX - f16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi 

Token type + network: FIL (Filecoin Network)
Deposit amount: 25
Deposit address: f16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi
TXID: bafy2bzacebfh6v3f57en2suofbc5rxkf7xx3oyv3p4p43e2povtdneqummygy

Transaction hash links:
[https://filfox.info/en/message/bafy2bzacebfh6v3f57en2suofbc5rxkf7xx3oyv3p4p43e2povtdneqummygy](https://www.google.com/url?q=https%3A%2F%2Ffilfox.info%2Fen%2Fmessage%2Fbafy2bzacebfh6v3f57en2suofbc5rxkf7xx3oyv3p4p43e2povtdneqummygy&sa=D&sntz=1&usg=AOvVaw3TjUfm4qWozNC43moCQBsD)

[https://beryx.zondax.ch/v1/search/fil/mainnet/transactions/bafy2bzacebfh6v3f57en2suofbc5rxkf7xx3oyv3p4p43e2povtdneqummygy](https://www.google.com/url?q=https%3A%2F%2Fberyx.zondax.ch%2Fv1%2Fsearch%2Ffil%2Fmainnet%2Ftransactions%2Fbafy2bzacebfh6v3f57en2suofbc5rxkf7xx3oyv3p4p43e2povtdneqummygy&sa=D&sntz=1&usg=AOvVaw1ExcUIrpUaD8UsT5v3qaCU)

[https://explorer.glif.io/tx/0x532778d48578035bdd43f026bb4d2f0099227195c5dc2858a0982ad9da4f74b4/](https://www.google.com/url?q=https%3A%2F%2Fexplorer.glif.io%2Ftx%2F0x532778d48578035bdd43f026bb4d2f0099227195c5dc2858a0982ad9da4f74b4%2F&sa=D&sntz=1&usg=AOvVaw1t_TDTEm3ao54Ar-mliPzh)



Explorer link for address:
1. [[https://filfox.info/en/address/f16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi](https://www.google.com/url?q=https%3A%2F%2Ffilfox.info%2Fen%2Faddress%2Ff16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi&sa=D&sntz=1&usg=AOvVaw3nhbBd5a5SqAQ6jsZsMEzx)](https://filfox.info/en/address/f16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi)

2. [https://beryx.zondax.ch/v1/search/fil/mainnet/address/f16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi](https://www.google.com/url?q=https%3A%2F%2Fberyx.zondax.ch%2Fv1%2Fsearch%2Ffil%2Fmainnet%2Faddress%2Ff16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi&sa=D&sntz=1&usg=AOvVaw02rvKdGRhLrWFVBRqz9ETM)

3. [https://explorer.glif.io/address/f16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi/](https://www.google.com/url?q=https%3A%2F%2Fexplorer.glif.io%2Faddress%2Ff16o6ueawep6xaei7fkytsd6nbsj3iuryqu2euevi%2F&sa=D&sntz=1&usg=AOvVaw29JjxxpceCC_AqWc6VBFgI)


OKX Customer Support response:
Greeting from OKX support!
 
Upon checking, you have used a method that we don't support.

Please be advised that we have recorded and escalated the deposit for further review.
 
However, due to the complexity of the situation, we cannot provide a definitive timeframe as to when this will be credited to your account.
 
Should you have any concerns, kindly contact us via chat or email anytime. Have a great day ahead!

**Reply by MattCrBoom** [reply]
johh!

**Comment by adaOctopus** [comment]
Hello! Something strange was going on this week with Binance when sending FILECOIN to it.
First of all, it took way more than usual to see the Filecoin i transferred to Binance than usual (but probably that's Binance's false)
Secondly, the FIL remained as "frozen" for 8 HOURS. It was shown as FROZEN in Binance and could not be withdrawn. It required 900 block confirmations for me to be able to take them out. That's absurd. Is this a binance fault or something going on with block confirmations?
Im raising this issue, because due to the volatility of Crypto, 8 hours in crypto is like 6 months in real time units :)

**Comment by banamicobo** [comment]
I have sent FIL from 0xfecc2cd80f73e00576767474c62f60802bb4f5f5 (metamask) to 0x0B573dA5715A88d9474E5ab07068D94379862A1b (binance) and my funds got stuck under placeholder https://filscan.io/en/address/f410fbnlt3jlrlkensr2olkyha2gzin4ymkq37hbseua/ and https://filscan.io/en/address/f410fayn5ow24fnktmz6tsru5qa3um4mfpmvwuc5rbyy/. Binance claim that they not in control of the fund since f410fayn5ow24fnktmz6tsru5qa3um4mfpmvwuc5rbyy does not belong to binance. My funds are stuck (6 month already).

**Comment by VenusOfficial** [comment]
👀

**Comment by MattCrBoom** [comment]
nfn
